fbw-a32nx/src/wasm/fbw_a320/src/interface/SimConnectInterface.cpp

```diff
@@ -1802,6 +1891,44 @@ void SimConnectInterface::processEvent(const DWORD eventId, const DWORD data0, c
       break;
     }
 
+    case Events::A32NX_SECONDARY_ELEVATOR_SET: {
+      simInput.secondaryInputs[AXIS_ELEVATOR_SET] = static_cast<long>(data0) / 16384.0;
+      if (loggingFlightControlsEnabled) {
+        std::cout << "WASM: SECONDARY_ELEVATOR_SET: ";
+        std::cout << static_cast<long>(data0);
+        std::cout << " -> ";
+        std::cout << simInput.secondaryInputs[AXIS_ELEVATOR_SET];
+        std::cout << std::endl;
```

Comment:
These logs will absolutely spam the console. Do we really need them? If so make sure to print them properly so they only produce one message rather than one per line as they currently do.

Perhaps they should all be behind an ifdef DEBUG_EVENTS or something.

Reply:
Well they are guarded by the `loggingFlightControlsEnabled` check, which is an LVar that can be set or also configured by INI file, it defaults to false. So I don't see a problem here